src/commands/autobump-threads/add-thread.test.ts

```diff
@@ -37,10 +37,7 @@ describe('Add autobump thread', () => {
       id: threadId,
       type: ChannelType.PublicThread,
     } as unknown as TextChannel);
-    mockAddAutobumpThread.mockResolvedValueOnce({
-      success: false,
-      error: new Error('Synthetic Error'),
-    });
+    mockAddAutobumpThread.mockRejectedValueOnce(new Error('Synthetic Error'));
```

Comment:
Since we don't have to return `OpPromise` or `OpPromise` anymore, this will simplify the mock.